### ISSUE TITLE
Each build needs to pull the image

### ISSUE THREAD

**Issue by akkuman** [crazy-max/ghaction-xgo#224]
Each build needs to pull the image

![image](https://user-images.githubusercontent.com/7813511/137710861-c6b8de05-b99a-43a9-85a9-8a3f4fe0717f.png)

**Comment by crazy-max** [comment]
Yes this is expected. Maybe we could cache the image so it would not need to be pulled on every run.
